bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigUtil.java

```diff
@@ -138,8 +135,10 @@ public static Object normalizeType(Object value, ConfigDescriptionParameter conf
         for (Entry<String, ?> parameter : configuration.entrySet()) {
             String name = parameter.getKey();
             Object value = parameter.getValue();
-            ConfigDescriptionParameter configDescriptionParameter = configParams.get(name);
-            convertedConfiguration.put(name, normalizeType(value, configDescriptionParameter));
+            if (!name.equals("objectClass") && !name.equals("component.name") && !name.equals("component.id")) {
```

Comment:
Now that I'm seeing this, I think it would be good to have a static set of strings defined once, then checking here (and above) only if the name is contained in there - just to make sure they don't run out of sync.